libkmod/libkmod.c

```diff
@@ -802,8 +802,15 @@ KMOD_EXPORT int kmod_dump_index(struct kmod_ctx *ctx, enum kmod_index type, int
 	if (ctx == NULL)
 		return -ENOSYS;
 
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wtautological-unsigned-enum-zero-compare"
+#endif
```

Comment:
wondering if we should use a _Pragma so we can hide where it's available in a define somewhere out of sight.

Reply:
maybe when we have one more case...